### PR TITLE
LPS-149255 Don't show paid and organic endpoint.

### PR DESCRIPTION
**Pull request by cgoncas**
Motivation
=======
We are going to remove the Semrush support from Liferay
[Link to story or ticket](https://issues.liferay.com/browse/LPS-149255)

Proposed Solution
========
If the  feature.flag.LPS-149255=true (defined in portal-ext.properties), no endpoint are shown for Organical or Paid Traffic Channels.

Steps to Verify:
----------------
1. Open the Content Performance Panel for a Page with Social and Paid TS.
1. Check that the enpoint is not shown.

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/TrafficSources.js [hunk 1]
```diff
@@ -173,8 +173,10 @@ export default function TrafficSources({dataProvider, onTrafficSourceClick}) {
 						<tbody>
 							{trafficSources?.map((entry) => {
 								const hasDetails =
-									entry?.value > 0 && entry.name !== 'direct';
-
+									entry?.value > 0 &&
+									(entry.name === 'referral' ||
```
Comment (cgoncas) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/TrafficSources.js:177]: Hey @Stephy6 and @claraizquierdo , what do you think about using a more generic approach, like if the endpoint is available or not?

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci.test:relevant

**Comment by cgoncas** [comment]
ci:test:tango

**Comment by cgoncas** [comment]
Hey @Stephy6 and @claraizquierdo !!

source-formatter needs to be run in the JS of this PR.

Thanks!!

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Review by claraizquierdo — APPROVED** [review]
LGTM! I just squashed two commits because they has the same commit message.

**Comment by cgoncas** [comment]
ci:forward

**Pull request closed without merging**